src/eva/vision/utils/convert.py

```diff
@@ -6,19 +6,16 @@
 import numpy.typing as npt
 
 
-def to_8bit(image_array: npt.NDArray[Any]) -> npt.NDArray[np.uint8]:
-    """Casts an image of higher bit image (i.e. 16bit) to 8bit.
+def to_8bit(image: npt.NDArray[Any]) -> npt.NDArray[np.uint8]:
+    """Casts an image array of higher bit image (i.e. 16bit) to 8bit.
 
     Args:
-        image_array: The image array to convert.
+        image: The image array to convert.
 
     Returns:
-        The image as normalized as a 8-bit format.
+        The image as uint8 (0, 255) array.
     """
-    if np.issubdtype(image_array.dtype, np.integer):
-        image_array = image_array.astype(np.float64)
-
-    image_scaled_array = image_array - image_array.min()
-    image_scaled_array /= image_scaled_array.max()
-    image_scaled_array *= 255
-    return image_scaled_array.astype(np.uint8)
+    image_scaled = image - image.min()
```

Comment:
This logic seems similar to `_denorm_image` function?

Reply:
its exactly the same yes, but this one is for numpy arrays and the other for tensor - i didn't find a way to combine them without covering an image two times (tensor -> array -> tensor) 🤔

Reply:
Good point. At least for the min-max scaling, something like this works as torch & numpy share the same API for min/max:
```
def min_max_scale(image: npt.NDArray[Any] | torch.Tensor]) -> npt.NDArray[Any] | torch.Tensor:
    return (image - image.min()) / (image.max() - image.min())
```

The issue with that of course would be that the return type isn't uniquely defined. Could be solved maybe with something like `@functools.singledispatch`. But then going there is probably overkill for this case..
Can also leave it as is.

Reply:
actually, after conversation with radiology lab, I removed completely the numpy version, as the 8-bit range is not interested for them

So I moved the tensor implementation to public `conver` file

Reply:
perfect, thanks!